### PR TITLE
Augmented tensor: labels renamed by Child + default args when adding one node

### PR DESCRIPTION
**Pull request by thibo73800**
In this merge request:
- Rename `label` in augmented tensor by `child` & children (datasets samples need to be updated)
- Default mergeable & align_dim when adding node

Example
```python
import aloscene
import torch

frame = aloscene.Frame("/path/to/image.jpg")

vector = torch.as_tensor([0, 1, 2])
# align on T & B by defautl & mergeable
vector = aloscene.tensors.AugmentedTensor(vector, names=(None,))
frame.add_child("vector", vector)

frame2 = frame.resize((200, 200))
frame = frame.batch_list([frame, frame2])
frame.get_view().render()
```

@ragier 
@jsalotti 
@Johansmm 
@LucBourrat1 

@Johansmm  Can you try out the changes on your current project and let me know the errors you stil lhave (with some simple code to reproduce)

### CODE REVIEW

File: aloscene/bounding_boxes_2d.py [hunk 1]
```diff
@@ -77,17 +77,17 @@ def boxes2xcyc(tensor):
             return tensor
         elif tensor.boxes_format == "xyxy":
             # Convert from xyxy to xcyc
-            labels = tensor.drop_labels()
+            labels = tensor.drop_childs()
```
Comment (ragier) [inline aloscene/bounding_boxes_2d.py:80]: childs -> children
